src/commcare_cloud/commands/ansible/run_module.py

```diff
@@ -136,8 +138,8 @@ def run_ansible_module(ansible_context, inventory_group, module, module_args,
 
     env_vars = ansible_context.build_env(need_secrets=become)
     if run_command is ansible_json:
-        env_vars.setdefault("ANSIBLE_LOAD_CALLBACK_PLUGINS", "1")
-        env_vars.setdefault("ANSIBLE_STDOUT_CALLBACK", "json")
+        env_vars["ANSIBLE_LOAD_CALLBACK_PLUGINS"] = "1"
+        env_vars["ANSIBLE_STDOUT_CALLBACK"] = "json"
```

Comment:
To check my understanding. The idea here is what you described as

> - Always set callback environment variables (it would break if they were not set properly).

i.e. the command these env vars feed into will break if `ANSIBLE_LOAD_CALLBACK_PLUGINS` and `ANSIBLE_STDOUT_CALLBACK` are set to anything other than these values, and so giving the caller the ability to (accidentally) override them is just asking for trouble without providing any value. Is that right?

Reply:
Correct.